common/test/src/IO/CompilationConfigParserTest.cpp

```diff
@@ -248,6 +248,29 @@ TEST_CASE(
   CHECK_THROWS_AS(parser.parse(), ParserException);
 }
 
+TEST_CASE(
+  "CompilationConfigParserTest.parseOneProfileWithNameAndOneDeleteTaskWithMissingTarget",
+  "[CompilationConfigParserTest]")
+{
+  const std::string config(
+    "{\n"
+    "    'version': 1,\n"
+    "    'profiles': [\n"
+    "        {\n"
+    "             'name': 'A profile',\n"
+    "             'workdir': '',\n"
+    "             'tasks': [\n"
+    "                 {\n"
+    "                      'type':'delete',\n"
+    "                 }\n"
+    "             ]\n"
+    "        }\n"
+    "    ]\n"
+    "}\n");
+  CompilationConfigParser parser(config);
```

Comment:
Here too: `auto parser = CompilationConfigParser{config};` and for all other declarations in the tests you added.